.mergify.yml

```diff
@@ -1,13 +1,43 @@
+# This file was automatically generated by sbt-typelevel-mergify using the
+# mergifyGenerate task. You should add and commit this file to
+# your git repository. It goes without saying that you shouldn't edit
+# this file by hand! Instead, if you wish to make changes, you should
+# change your sbt build configuration to revise the mergify configuration
+# to meet your needs, then regenerate this file.
+
 pull_request_rules:
-- name: automatically merge scala-steward's PRs
-  actions:
-    merge:
-      method: squash
+- name: merge scala-steward's PRs
   conditions:
   - author=scala-steward
-  - body~=labels:.*semver-patch.*
+  - body~=labels:.*early-semver-patch
   - status-success=Build and Test (ubuntu-latest, 2.12.15, temurin@11)
   - status-success=Build and Test (ubuntu-latest, 2.13.8, temurin@11)
   - status-success=Build and Test (ubuntu-latest, 2.13.8, temurin@17)
   - status-success=Build and Test (ubuntu-latest, 3.1.1, temurin@11)
-  - status-success=Generate Site (ubuntu-latest, 2.13.8, temurin@11)
+  actions:
+    merge:
+      method: squash
+- name: Label core PRs
+  conditions:
+  - files~=^core/
+  actions:
+    label:
+      add:
+      - core
+      remove: []
+- name: Label docs PRs
+  conditions:
+  - files~=^docs/
+  actions:
+    label:
+      add:
+      - docs
+      remove: []
```

Comment:
The plugin also sets up auto-labeling, hope that's okay. Can't turned it off because of a bug 😅 
- https://github.com/typelevel/sbt-typelevel/issues/220

Reply:
Not a blocker for this one, but thanks for the heads up